lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -963,6 +1001,10 @@ WasmBinaryReader::ReadImportEntries()
         {
             WasmTypes::WasmType type = ReadWasmType(len);
             bool isMutable = ReadConst<UINT8>() == 1;
+            if (isMutable)
```

Comment:
Different issue, but noticed it now. We should also check for other values and throw decoding error (i.e. if it is anything other than 0 or 1).